loadgen/cmd/otelsoak/config.example.yaml

```diff
@@ -1,5 +1,6 @@
 receivers:
   loadgen:
+#    concurrency: 16  # defaults to GOMAXPROCS
```

Comment:
Out of curiosity, how is `otelsoak` configured to stop at some point? I don't see a possibility to configure a time or similar?

Reply:
It does not terminate now. I'm not aware of a way to notify the otel collector to stop, unless we write code to make it panic or os.Exit

Reply:
ok, thanks for confirming; probably something for us to look into, but in another issue/PR